### ISSUE TITLE
Installing from pypi fails

### ISSUE THREAD

**Issue by SanRehmo** [wayfair-incubator/extra-model#38]
installing extra-model from pypi fails with the following error message:

```Collecting cymem<2.1.0,>=2.0.2 (from spacy==2.0.18->extra-model)
  Downloading https://files.pythonhosted.org/packages/3e/fb/5899a59ee8d0f02202c1f02fe47671e0c93d1812b1deb2491505718473da/cymem-2.0.5.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/wayfair/tmp/pip-build-88wt78b9/cymem/setup.py", line 10, in <module>
        from Cython.Build import cythonize
    ModuleNotFoundError: No module named 'Cython'
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /wayfair/tmp/pip-build-88wt78b9/cymem/
```

System information:

OS: `centos 7.5`
Python version: `3.6.0`
Command run: `pip install --index-url https://pypi.org/simple extra-model`

Running `pip install cython` manually before attempting to install `extra-model` fixed this. It might be an issue with our `setup.py` file: https://github.com/SciTools/cf-units/issues/106

**Comment by chrisantonellis** [comment]
Thanks for the bug report! I will attempt to replicate the issue and report back!

**Comment by chrisantonellis** [comment]
I'm unable to replicate the issue. @SanRehmo Can you provide more specific instructions on how to replicate? Heres what I tried...

1. `docker run -it --entrypoint bash wayfair/python/centos75-python36-devbox:3.6.11-2020.11.05`
2. `pip install extra-model==0.2.0rc0`

thank you

**Comment by SanRehmo** [comment]
Huh, odd. I also can't reproduce it anymore in the same venv where the error first appeared. I've tried to uninstall both `extra-model` and `cython` and reinstall `extra-model` again and now it works flawlessly.

I also tried switching to a fresh venv however and there the same error gets thrown when running `pip install extra-model`. That venv is also running Python `3.6.0`. Here's the output of `pip list` for that venv:

```
pip (9.0.1)
setuptools (28.8.0)
```

After upgrading `pip` to version `21.0.1`, `extra-model` could be installed without a problem.

I verified this by creating another fresh venv, upgrading `pip` and installing `extra-model`, which worked fine as well.

So the problem seems to be an outdated `pip` version somehow breaking the `cymem` installation.
